brain/python/test.cpp

```diff
@@ -18,9 +16,29 @@
  * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
  */
 
-#ifndef BRAIN_BINDING_SPIKEREPORTWRITER_H
-#define BRAIN_BINDING_SPIKEREPORTWRITER_H
+#ifndef BRION_USE_BBPTESTDATA
+void export_test() {}
+#else
+
+#include <boost/python.hpp>
+
+#include "submodules.h"
+
+#include <BBP/TestDatasets.h>
+
+using namespace boost::python;
+
+void export_test()
+{
+    list configs;
+    for( const auto& config : bbp::test::getBlueconfigs( ))
+        configs.append( config );
+
+    scope test = brain::exportSubmodule("test");
 
-void export_SpikeReportWriter();
+    test.attr("blue_config") = bbp::test::getBlueconfig();
```

Comment:
one should really update the test data repo to not report that super-old circuit from may_2011 here (in C++, I faked it by using the BBP_TEST_BLUECONFIG3 string define)